### DISCUSSION TITLE
show filters above table: function mentioned in documentation not working?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#5128] [Help] [answered]
I set the filters above the table using:
```
            ->filters([
                //
            ], layout: Layout::AboveContent)
```

This because the [function mentioned in the documentation](https://filamentphp.com/docs/2.x/tables/filters#displaying-filters-above-or-below-the-table-content) did not have any effect: 

```
protected function getTableFiltersLayout(): ?string
{
    return Layout::AboveContent;
}
```
Is that the intended behavior?

Using v2.16.52.

**Comment by danharrin** [comment, marked as answer]
If you're overriding the method, it needs to be on the list page, not the resource.

**Comment by pechtelt1** [comment]
@wivaku Do you know if it is possible to add some filters in the pop-up? And some filters AboveContent?
